draft-ietf-uuidrev-rfc4122bis.md

```diff
@@ -1512,8 +1518,79 @@ and feedback.
 
 
 # IANA Considerations {#IANA}
-TODO: Q: Should Namespace Registration Template be here or in {{namespace_reg_template}}?
-TODO: Need to ensure IANA doc, https://www.iana.org/assignments/urn-namespaces/urn-namespaces.xhtml, has this new document listed.
+
+Per {{RFC8141}} here is the Namespace Registration Template filled out for this namespace and hereby a request to reference this document (when the final version is published) at {{URNNamespaces}}.  Note that namespace is already listed and this is a request to update that entry to reference this document.
+
+Namespace Identifier: UUID (formal)
+
+Version: 1
+
+Date: 2003-10-01
+
+Registrant: JTC 1/SC6 (ASN.1 Rapporteur Group)
+
+Purpose: A UUID is an identifier that is unique across both space and time,
+  with respect to the space of all UUIDs.  Since a UUID is a fixed
+  size and contains a time field, it is possible for values to
+  rollover (around A.D. 3400, depending on the specific algorithm
+  used).  A UUID can be used for multiple purposes, from tagging
+  objects with an extremely short lifetime, to reliably identifying
+  very persistent objects across a network.
+
+Syntax: The internal representation of a UUID is a specific sequence of
+  bits in memory, as described in {{format}}.  To accurately
+  represent a UUID as a URN, it is necessary to convert the bit
+  sequence to a string representation.
+
+  Each field is treated as an integer and has its value printed as a
+  zero-filled hexadecimal digit string with the most significant
+  digit first.  The hexadecimal values "a" through "f" are output as
+  lower case characters and are case insensitive on input.
+
+  The formal definition of the UUID string representation is
+  provided by the following ABNF {{RFC4234}}:
```

Comment:
```suggestion
  provided by the following ABNF {{RFC5234}}:
```

Reply:
Maybe the ABNF could be referenced to this document, rather than repeated in the template.